Automattic/wp-calypso
# Inline Help: float the help button in the lower right

This PR moves the help button ("?") from the sidebar footer into the lower right and makes it a floating element. We hope this improves the visibility of help in Calypso. 

Screen capture: 

![inline-help-float-right](https://user-images.githubusercontent.com/23619/37520378-081cadd0-291d-11e8-99ce-7cefaaa9a3c3.gif)

To test: 
- check whether the floating icon is visible and works well on different pages of Calypso — e.g., http://calypso.localhost:3000/stats / http://calypso.localhost:3000/help / 
- make sure this looks and works well under a variety of screen resolutions and devices
